packages/mdc-toolbar/README.md

```diff
@@ -40,6 +40,20 @@ npm install @material/toolbar
 
 ## Usage
 
+### Menu Icon
```

Comment:
This technically shouldn't be up here (though this readme isn't following the new standard and we shouldn't worry about that since this package is deprecated)...  could this work as a subheading under HTML structure instead at least?